ios/Capacitor/Capacitor/CAPBridgeViewController.swift

```diff
@@ -160,6 +162,24 @@ public class CAPBridgeViewController: UIViewController, CAPBridgeDelegate, WKScr
 
   }
 
+  public func setStatusBarDefaults() {
+    if let url = Bundle.main.url(forResource:"Info", withExtension: "plist"),
+      let plist = NSDictionary(contentsOf: url) as? [String:Any] {
```

Comment:
You can get the info plist information with `let plist = Bundle.main.infoDictionary`.

Other than that, looks good to me.

Might be good to also document this in the Statusbar plugin section

Reply:
Changes made and added a line to the status bar docs!